README.md

```diff
@@ -1,5 +1,18 @@
 # TubiPlayer
-A media streaming video player based on ExoPlayer from google, with convenience methods for ad supported content
+The official android player of [Tubi TV](https://www.tubi.tv) -- A free movie and TV streaming service.
 
+# Purposes of TubiPlayer
+In the industry of streaming free multi-media contents to users, companies choose to show In-stream video ad to user between the main content as their business model. Examples: **YouTube**, **Tubi TV**.
```

Comment:
In the industry of streaming free multi-media contents to users, companies choose to show in-stream video ads to users between the main content as their business model. Examples include YouTube and Tubi TV.